### DISCUSSION TITLE
Ajout Onduleur Sofar Solar

### DISCUSSION THREAD

**Discussion by strullutony** [slipx06/sunsynk-power-flow-card#293] [Ideas]
Bonjour

Serait il possible d'ajouter les onduleurs Sofar Solar dans les images de présentation "modèle HYD5000EP"

Merci

**Comment by strullutony** [comment]
![onduleur-hybride-6kw-dual-mppt-hyd-6000-ep-sofar-sofar-solar](https://github.com/slipx06/sunsynk-power-flow-card/assets/64546939/edf6cf6f-5699-4941-bfee-5fbcff33ce15)

**Reply by slipx06** [reply]
I'll take a look

**Comment by slipx06** [comment]
I've added image for this inverter to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.21.0

**Reply by strullutony** [reply]
Merci super travail

Pour ma connaissance personnel, j'ai essayé de modifier le code pour changer l'image. Même en intégrant une image en base 64 bien dimensionnée, Home assistant reste sur l'ancienne image.
J'ai modifié le code du fichier sunsynk-power-flow-card.js. Faut il modifier l'image a un autre endroit ?
Y a t'il quelque chose à recharger dans HA ?

**Reply by slipx06** [reply]
There has been a major refactor to the code so this has changed now, but you can review the commit and files changed here https://github.com/slipx06/sunsynk-power-flow-card/commit/e779f12949bb1fd098678088980ef82508f21547
